### PR TITLE
Add steps for setting up Codespaces

### PR DESCRIPTION
**Pull request by guenp**
Adding instructions on how to set up Visual Studio Codespaces.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/54b722b40e1652e80392072909513c35978f1e1a/articles/quickstarts/install-command-line.md) | 82 | 67 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b73c54d5-ff88-4183-8d5a-900822ec2bc4) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by guenp** [comment]
@KittyYeungQ  not sure how to add you as reviewer but would appreciate if you could take a look :) Thanks!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2b494d89d51486305ca00b3455de5e49a6630f14/articles/quickstarts/install-command-line.md) | 82 | 67 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1c664535-3a02-4537-b228-ba71896bce6f) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/install-command-line.md [hunk 1]
```diff
@@ -18,7 +18,11 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+
+To configure Visual Studio Codespaces:
+1. Create an Azure account [here](https://azure.microsoft.com/en-us/free/).
+2. Create a Codespaces environment. Please follow the quickstart guide [here](https://docs.microsoft.com/visualstudio/online/quickstarts/browser). When creating the Codespace, make sure to enter microsoft/Quantum in the "Git Repository" field.
```
Comment (cgranade) [inline articles/quickstarts/install-command-line.md:26]: This makes sense for creating a Codespace from the samples repo, but do we want to provide any steps that users can follow to use VS Codespaces with their own projects?

File: articles/quickstarts/install-command-line.md [hunk 2]
```diff
@@ -18,7 +18,11 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+
+To configure Visual Studio Codespaces:
+1. Create an Azure account [here](https://azure.microsoft.com/en-us/free/).
```
Comment (cgranade) [inline articles/quickstarts/install-command-line.md]: I'd suggest dropping en-us for localization, and modifying the link text for SEO.
```suggestion
1. Create an [Azure account](https://azure.microsoft.com/free/).
```

File: articles/quickstarts/install-command-line.md (hunk 1 above)
Comment (guenp) [reply articles/quickstarts/install-command-line.md:26]: Yes that is a great point... We should probably make this more general.
Would it be an idea to create a template repo (like https://github.com/cgranade/quantum-research-template) and link that here?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4d1026f7bd9d7fbad20c19c0aa92cbfb21743337/articles/quickstarts/install-command-line.md) | 82 | 67 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49647fca-4bdd-4849-b6a3-c2aab57c48aa) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/install-command-line.md (hunk 1 above)
Comment (cgranade) [reply articles/quickstarts/install-command-line.md]: I think that could make a lot of sense, yeah. Probably beyond the scope of this PR, but something we can do fairly quickly afterwards.

Comment (guenp) [reply articles/quickstarts/install-command-line.md]: Sounds good, I'll add a note here to specify that any repo could be used, but we recommend `microsoft/Quantum` by default.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/51a9274b393bb6024db76510a54518b66c489148/articles/quickstarts/install-command-line.md) | 82 | 62 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/676bdef0-3cf4-4b92-8337-478b075970e1) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/install-command-line.md [hunk 3]
```diff
@@ -18,7 +18,12 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
```
Comment (KittyYeungQ) [inline articles/quickstarts/install-command-line.md:21]: Looks good overall. Do we want to make Codespaces the first option though, as that would give people an impression that is it promoted more as a default? Also we should briefly describe the difference between these environments (like this page does https://docs.microsoft.com/en-us/quantum/quickstarts/) especially given Codespaces requires an Azure account, would be good to call out the other two options do not require it.

Comment (guenp) [reply articles/quickstarts/install-command-line.md:21]: Great point! I've added a distinction as VS Code/Visual Studio being for local development and VS Codespaces as a cloud/browser option. I also linked on how to use VS Code to a Codespaces environment.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/489b731d5139c2a65291cf15464ebcead67c8c44/articles/quickstarts/install-command-line.md) | 83 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/137c043e-c8de-43fb-8c60-e05afb364fc0) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cd3facbb9b56d94f6067e1706f767b9556d67da2/articles/quickstarts/install-command-line.md) | 83 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f02c77ed-329f-4576-bed6-04e78ba4f4cb) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/quickstarts/install-command-line.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for developing your Q# applications locally. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more. For developing in the Cloud via the web browser, we recommend Visual Studio Codespaces.
```
Comment (KittyYeungQ) [inline articles/quickstarts/install-command-line.md:21]:
```suggestion
While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for developing your Q# applications locally. For developing in the Cloud via the web browser, we recommend Visual Studio Codespaces. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more. 
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a294c815a2943e705aefdd814f6d371520d69460/articles/quickstarts/install-command-line.md) | 83 | 64 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9606eeaa-b7c5-40e8-a1a1-9fc92658081d) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**